### PR TITLE
Make ACL login error message more generic

### PR DESCRIPTION
**Pull request by animesh2049**
In case of invalid login request we don't reveal info about user. So make error message more generic
Fixes DGRAPH-2468

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6716)

### CODE REVIEW

File: ee/acl/acl_test.go [hunk 1]
```diff
@@ -3107,3 +3107,29 @@ func TestMutationWithValueVar(t *testing.T) {
...
+	if err := grootClient.Alter(ctx, &op); err != nil {
+		t.Fatalf("Unable to cleanup db:%v", err)
+	}
+
+	client, err := testutil.DgraphClient(testutil.SockAddr)
+	require.NoError(t, err)
+
+	// User is not present
+	err = client.Login(ctx, userid, "simplepassword")
+	require.Error(t, err)
```
Comment (jarifibrahim) [inline ee/acl/acl_test.go:3126]: You can do `require.EqualError(t, err, x.ErrorInvalidLogin.Error())`

File: ee/acl/acl_test.go [hunk 2]
```diff
@@ -3107,3 +3107,29 @@ func TestMutationWithValueVar(t *testing.T) {
...
 	testutil.CompareJSON(t, `{"me": [{"name":"r1","nickname":"r1"}]}`, string(resp.GetJson()))
 }
+
+func TestFailedLogin(t *testing.T) {
+	ctx, _ := context.WithTimeout(context.Background(), 1*time.Second)
+
+	grootClient, err := testutil.DgraphClientWithGroot(testutil.SockAddr)
+	require.NoError(t, err)
+	op := api.Operation{DropAll: true}
+	if err := grootClient.Alter(ctx, &op); err != nil {
```
Comment (jarifibrahim) [inline ee/acl/acl_test.go:3117]: require.NoError(...)

File: ee/acl/acl_test.go [hunk 3]
```diff
@@ -3107,3 +3107,29 @@ func TestMutationWithValueVar(t *testing.T) {
...
+
+	// User is not present
+	err = client.Login(ctx, userid, "simplepassword")
+	require.Error(t, err)
+	require.Contains(t, err.Error(), x.ErrorInvalidLogin.Error())
+
+	resetUser(t)
+	// User is present
+	err = client.Login(ctx, userid, "randomstring")
+	require.Error(t, err)
```
Comment (jarifibrahim) [inline ee/acl/acl_test.go:3132]: You can do `require.EqualError(t, err, x.ErrorInvalidLogin.Error())`

File: x/x.go [hunk 4]
```diff
@@ -66,6 +66,8 @@ var (
 	// ErrNotSupported is thrown when an enterprise feature is requested in the open source version.
 	ErrNotSupported = errors.Errorf("Feature available only in Dgraph Enterprise Edition")
 	ErrNoJwt        = errors.New("no accessJwt available")
+	// ErrorInvalidLogin is error when username or password is incorrect in login
```
Comment (jarifibrahim) [inline x/x.go:69]:
```suggestion
	// ErrorInvalidLogin is returned when username or password is incorrect in login
```

File: edgraph/access_ee.go [hunk 5]
```diff
@@ -76,7 +76,7 @@ func (s *Server) Login(ctx context.Context,
 	if err != nil {
 		errMsg := fmt.Sprintf("Authentication from address %s failed: %v", addr, err)
 		glog.Errorf(errMsg)
```
Comment (jarifibrahim) [inline edgraph/access_ee.go:78]: `glog.Errorf(Authentication from address %s failed: %v", addr, err)`

**Review by jarifibrahim — APPROVED** [review]
Got some comments.

File: ee/acl/acl_test.go (hunk 1 above)
Comment (animesh2049) [reply ee/acl/acl_test.go:3126]: `err` string is  `rpc error: code = Unknown desc = invalid username or password`.  So equality doesn't hold between `err` and `x.ErrorInvalidLogin`

File: ee/acl/acl_test.go (hunk 3 above)
Comment (animesh2049) [reply ee/acl/acl_test.go:3132]: Same as above

**Pull request merged**